include/seqan3/alignment/matrix/trace_directions.hpp

```diff
@@ -45,12 +44,10 @@ namespace seqan3
 {
 //!\brief Enable bitwise operators for enum seqan3::detail::trace_directions.
 //!\ingroup alignment_matrix
+//!\implements enum_bitwise_operators
+//!\sa seqan3::add_enum_bitwise_operators<seqan3::detail::trace_directions> enables combining enum values.
```

Comment:
I have a feeling that this might be a better fit for the enum itself since we augment the enum with some operators. Can you double-check the look and feel?

Reply:
As described below, than it only appears in the Variable list.